media/js/base/site.js

```diff
@@ -64,10 +64,7 @@
             if (pf.indexOf("Mac") !== -1) {
                 return 'oldmac';
             }
-            if (pf === '' &&
-                    /Firefox/.test(ua) &&
-                    /Mobile|Tablet/.test(ua) &&
-                    !(/Android/.test(ua))) {
+            if (pf === '' && /Firefox/.test(ua)) {
```

Comment:
Do we also know for sure that `navigator.platform === ''` on Firefox OS TV? We only seem to have the UA string looking in the bug.